[Content] Dragonriders Mission (Dangerous Games Part 1) (#6948)
endless-sky/endless-sky: data/human/human missions.txt

event "fumbling mugger followup timer"

> mission "Dangerous Games 1"

Reviewer: Have you explained why you've numbered this mission?

			choice
				`	(Head over to "Mr. Karengo.")`
				`	(Leave.)`
					decline
			`	As you make your way over, you note that nearly everyone in the bar has the same drink you do. In the middle of the crowd, a large, heavily muscled, and tattooed man is gesticulating wildly, apparently telling a story.`
			choice
				`	(Go listen to the man.)`
				`	(Leave.)`
					decline
			`	"... so then he asks me, 'Have you ever ridden one?' And I look back at this crazy joithead and I say, 'You're a crazy joithead.' But he's got me thinking, y'know? Nobody has ever ridden one. Least, not nobody that could talk after." He lets out a sharp laugh, then raises his voice. "So I ask you all; who wants to join Karengo to make history? Who wants to ride a dragon? I'll supply the jetpacks, you supply the courage."`
			`	You've heard the stories of the dragons flying through Skymoot's skies as a child. Great beasts; graceful and dangerous, the apex predators of the air; they are strikingly similar to the dragons described in ancient legends. Skymoot is host to many dangerous creatures, and these living myths hold a place at the top of that list.`

> 			`	A cheer goes up from the men, especially the drunk ones. Karengo starts to hand out death and accident waivers and collect money for the expedition. Suddenly, his expression darkens as he cocks his head, listening to an earpiece. "Listen up!" he yells, "Seems my regular pilot got in the way of some less reputable folk, and my replacement pilot lost his balls this evening and can't find 'em! We need ourselves a ship and a captain to bring this lot to <stopovers>! Who's brave enough? Who else wants to make history?"`

Reviewer: ```suggestion
			`	A cheer goes up from the men, especially the drunk ones. Karengo starts to hand out death and accident waivers and collects money for the expedition. Suddenly, his expression darkens as he cocks his head, listening to an earpiece. "Listen up!" he yells, "Seems my regular pilot got in the way of some less reputable folk, and my replacement pilot lost his balls this evening and can't find 'em! We need ourselves a ship and a captain to bring this lot to <stopovers>! Who's brave enough? Who else wants to make history?"`
```

			choice
				`	"Captain <last>, at your service. We leave at dawn!"`
				`	(Stay silent.)`
			`	You remain silent as Karengo surveys the crowd. "What, nobody with a ship has the cojones to make history? I didn't know this bar was full of a bunch of sissies."`

> 			`	One captain in the back of the bar jumps up. "Mama didn't raise no sissy. I'll bring you all to Skymoot." A cheer rises up from the crowd and the captain is treated to several rounds of drinks. After the captain and all the passengers are sufficiently drunk, they saunter their way out of the bar, drunkenly singing about adventure as they go.`

Reviewer: ```suggestion
			`	One captain in the back of the bar jumps up. "Mama didn't raise no sissy! I'll bring you all to Skymoot." A cheer rises up from the crowd and the captain is treated to several rounds of drinks. After the captain and all the passengers are sufficiently drunk, they saunter their way out of the bar, drunkenly singing about adventure as they go.`
```

				decline
			`	After a cheer rises up from the crowd, you're treated to several rounds of drinks by your passengers; you sense any attempt to refuse would be ignored. You're unsure of how much you drank, but it was enough to make sure that you didn't remember. You wake up significantly after dawn with empty bottles and unconscious men strewn all over the ship's mess hall. Karengo peels himself off the floor and gives you a crooked smile, shielding his eyes from the light. "Alright, <last>," he says, "let's do this."`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying the passengers hasn't entered this system yet. Better depart and wait for it.`
	on stopover
		conversation
			`Karengo directs you down through the atmosphere of Skymoot, steering clear of the usual tourist areas and, you realize, of the local law enforcement. As you're putting down in a clearing halfway up a mountain, you hear an otherworldly screech and your ship lurches violently to one side. You would have been thrown to the floor if you hadn't been strapped in.`

> 			`	"Look lively, lads!" shouts Karengo, the only person still standing. "Your chariot awaits!" He double-checks the jetpacks on the backs of each of the eight men, or "Dragonriders," as they creatively branded themselves during their first night of the voyage, then opens the side hatch and hurls the Dragonriders out the door before jumping himself.`

Reviewer: ```suggestion
			`	"Look lively, lads!" shouts Karengo, the only person still standing. "Your chariot awaits!" He double-checks the jetpacks on the backs of each of the eight men, or "Dragonriders," as they creatively branded themselves during the first night of the voyage, then opens the side hatch and hurls the Dragonriders out the door before jumping himself.`
```

			choice
				`	(Get everyone out of there.)`
					goto leave
				`	(Take off and leave everybody else behind.)`
			label betrayal
			action
				fail
			`	The ship takes off with a roar at your touch. Between the cries of the dragons, the sound of the engines, and the air rushing past, you can barely hear the fear and betrayal in the voices of the men you've left behind.`
				decline
			label leave
			`	You turn on the external speakers. "We are leaving," you blast to the survivors. "Get on board."`
			`	The defeated Dragonriders need no further encouragement. They beeline for the hatch and you flicker the shield to let them on deck, where they collapse, wide-eyed and panting. Karengo isn't with them. You turn to ask about him, when a flicker of motion on a monitor catches your eye.`

> 			`	Karengo is there, and he's riding a dragon, gripping the ridge of scales around its neck. The dragon is frantically trying to get him off, but deft application of his jetpack is, for now, keeping him on top of the beast. He lets out wild screams of triumph as they soar through the sky and at last you can understand the appeal of this whole expedition. Your admiration is cut short by two catastrophic crashes on either side of the ship and accompanying sirens. The dragons are trying to get through.`

Reviewer: ```suggestion
			`	Karengo is riding a dragon, gripping the ridge of scales around its neck. The dragon frantically tries to get him off, but deft application of his jetpack keeps him on top of the beast for now. He lets out wild screams of triumph as they soar through the sky and at last you can understand the appeal of this whole expedition. Your admiration is cut short by two catastrophic crashes on either side of the ship and accompanying sirens. The dragons are trying to get through.`
```

	on complete
		payment 12000 500
		conversation
			`As you land and fuel up, Karengo herds the seven surviving men out of the ship. "Meet us at the bar, yeah?" he says to you. "This day deserves a celebration." You nod, but can't help but notice red eyes and wet cheeks among the rest of the Dragonriders as they shuffle off.`
			`	Later that day, you spot the Dragonriders at a corner table of the bar. The men, with the exception of Karengo, appear subdued.`
			`	"How goes it, <last>?" he asks. "We were just raising a glass to mission success - made possible by you, of course. Great flying, Captain." The bartender arrives with a round of drinks and Karengo raises one. "To the Dragonriders!" he says.`
			choice
				`	"To the Dragonriders!"`
				`	"To our fallen friends!"`
				`	"I'm good, thanks."`

> 			`	Karengo gives you an unreadable look and lowers his glass.`
> 			`	"Your payment, plus Bolek and Jarek's signup fees: <payment>."`

Reviewer: ```suggestion
			`	Karengo gives you an unreadable look, lowers his glass, and says, "Your payment, plus Bolek and Jarek's signup fees: <payment>."`
```

			choice
				`	"Thanks."`
					goto end
				`	I can't accept their money. It's not right."`
			`	Karengo shrugs and doesn't take it back.`
			label end

> 			`	"Maybe I'll see you again sometime, yeah? I'm in need of a pilot with some guts." He nods in your direction, clearly dismissing you.`

Reviewer: ```suggestion
			`	"Maybe I'll see you again sometime, yeah? I'm always in need of a pilot with some guts." He nods in your direction, clearly dismissing you.`
```

event "fumbling mugger followup timer"

mission "Dangerous Games 1"
	name "Dragonriders"

> 	description "A lively man named Karengo and a few passionate youth are traveling with you to see the dragons on <stopover>. Take the group to <destination> and return the group back to <source> once they've finished."

Reviewer 2: ```suggestion
	description "A lively man named Karengo and a few passionate youth are traveling with you to see the dragons on <stopover>. Return the group back to <source> once they've finished."
```

Reviewer 2: This, too.

	stopover Skymoot
	on offer
		conversation
			`Upon entering the spaceport, you notice that the local bar seems to be more active than usual. The moment you pass through the bar door, a strong drink is pressed into your hand. You look at it, dumbfounded, and then look up to the waiter who gave it to you. "Compliments of Mr. Karengo," he says, nodding his head towards a crowd of people, mostly young men, before moving on.`
			choice
				`	(Head over to "Mr. Karengo").`
				`	(Leave).`
					decline
			`	As you make your way over, you note that nearly everyone in the bar has the same drink you do. In the middle of the crowd, a large, heavily muscled, and tattooed man is gesticulating wildly, apparently telling a story.`
			choice
				`	(Go listen to the man).`
				`	(Leave).`
					decline
			`	"...so then he asks me, have you ever ridden one? And I look back at this crazy joithead and I say, you're a crazy joithead. But he's got me thinking, y'know? Nobody has ever ridden one. Least, not nobody that could talk after." He lets out a sharp laugh, then raises his voice. "So I ask you all; who wants to join Karengo to make history? Who wants to ride a dragon?"`

> 			`	You've heard the stories of the dragons flying through <planet>'s skies as a child. Great beasts; graceful and dangerous, the apex predators of the air; they are strikingly similar to the dragons described in ancient legends. <planet> is host to many dangerous creatures, and these living myths hold a place at the top of that list.`

Reviewer 2: I don't think `<planet>` will reference the stopover planet, I'm fairly sure you'll just have to put the name here directly.

```suggestion
			`	You've heard the stories of the dragons flying through Skymoot's skies as a child. Great beasts; graceful and dangerous, the apex predators of the air; they are strikingly similar to the dragons described in ancient legends. Skymoot is host to many dangerous creatures, and these living myths hold a place at the top of that list.`
```

event "fumbling mugger followup timer"

mission "Dangerous Games 1"
	name "Dragonriders"

> 	description "A lively man named Karengo and a few passionate youth are traveling with you to see the dragons on <stopover>. Take the group to <destination> and return the group back to <source> once they've finished."

Reviewer 2: Should this say 'youths'?

Also, as far as I can tell, the source and destination for this mission will be the same?

Author: I tried to create a situation where you would pick them up at the origin planet, drop them off at Skymoot, and then fly them back to the origin planet. I used stopover for this, but I might not have used it correctly.

Reviewer 2: Yes, this mission should have you go to Skymoot and then back to where you started. `<destination>` is the final destination, which is in this case the source as well. So saying to take them to the destination and then back the the source makes no sense. You want to tell the player to take them to the stopovers and then to the destination/source.

Author: I tried fixing it, is it ok now?

Reviewer 2: You can mark this as resolved.

	minor
	passengers 9
	to offer
		random < 15
	source 
		attributes frontier pirate rim
	stopover Skymoot
	on offer
		conversation

> 			`	Upon entering the spaceport, you notice that the local bar seems to be more active than usual. The moment you pass through the bar door, a strong drink is pressed into your hand. You look at it, dumbfounded, and then look up to the waiter who gave it to you. "Compliments of Mr. Karengo," he says, nodding his head towards a crowd of people, mostly young men, before moving on.`

Reviewer 3: ```suggestion
			`Upon entering the spaceport, you notice that the local bar seems to be more active than usual. The moment you pass through the bar door, a strong drink is pressed into your hand. You look at it, dumbfounded, and then look up to the waiter who gave it to you. "Compliments of Mr. Karengo," he says, nodding his head towards a crowd of people, mostly young men, before moving on.`
```

			choice
				`	(Head over to "Mr. Karengo").`
				`	(Leave).`
					decline
			`	As you make your way over, you note that nearly everyone in the bar has the same drink you do. In the middle of the crowd, a large, heavily muscled, and tattooed man is gesticulating wildly, apparently telling a story.`
			choice
				`	(Go listen to the man).`
				`	(Leave).`
					decline
			`	"...so then he asks me, have you ever ridden one? And I look back at this crazy joithead and I say, you're a crazy joithead. But he's got me thinking, y'know? Nobody has ever ridden one. Least, not nobody that could talk after." He lets out a sharp laugh, then raises his voice. "So I ask you all; who wants to join Karengo to make history? Who wants to ride a dragon?"`

> 			`	You've heard the stories of the dragons flying through <planet>'s skies as a child. Great beasts; graceful and dangerous, the apex predators of the air; they are strikingly similar to the dragons described in ancient legends. <planet> is host to many dangerous creatures, and these living myths hold a place at the top of that list.`

Reviewer 2: ```suggestion
			`	A cheer goes up from the men, especially the drunk ones. Karengo starts to hand out death and accident waivers and collect money for the expedition. Suddenly, his expression darkens as he cocks his head, listening to an earpiece. "Listen up!" he yells, "Seems my regular pilot got in the way of some less reputable folk, and my replacement pilot misplaced his balls this evening and can't find 'em! We need ourselves a ship and a captain to bring this lot to <stopovers>! Who's brave enough? Who else wants to make history?"`
```

This'll become: `Skymoot in the Hadar system`, assuming I remember right and Hadar is the correct system, I'm not entirely sure. It might not appear right that he'd say it like this, so may e just `Skymoot` is better.

Author: It appears that you've made two comments on one line. Which one should I commit?

Reviewer 2: Ignore this one.

> 			`	A cheer goes up from the men, especially the drunk ones. Karengo starts to hand out death and accident waivers and collect money for the expedition. Suddenly, his expression darkens as he cocks his head, listening to an earpiece. "Listen up!" he yells, "Seems my regular pilot got in the way of some less reputable folk, and my replacement pilot misplaced his balls this evening and can't find 'em! We need ourselves a ship and a captain to bring this lot to <stopovers>! Who's brave enough? Who else wants to make history?"`

Reviewer 3: `...my replacement pilot misplaced his balls this evening and can't find 'em!`

Seems inappropriate.

			choice

> 				`	"Captain <last>, at your service. We leave at dawn!"`

Reviewer 2: This should lead either to `decline` or something about how you leave them to their activities and then decline.

Reviewer 2: This shouldn't be on this line. This review really got fucked.

				`	(Stay silent)."`
					decline

> 			`	You wake up significantly after dawn with empty bottles and unconscious men strewn all over the ship's mess hall. Karengo peels himself off the floor and gives you a crooked smile, shielding his eyes from the light. "Alright, <last>" he says, "let's do this."`

Reviewer 3: This timeskip comes seemingly out of nowhere. Was there no immediate reaction to a captain within the crowd agreeing to the transport? No cheering or similar celebration at the player's willingness to help Karengo?

			on visit
				dialog `You land on <planet>, but realize that your escort carrying the passengers hasn't entered this system yet. Better depart and wait for it.`
			on stopover

> 				conversation

Reviewer 2: ```suggestion
					`The men load into a shuttle and Karengo directs you down through the atmosphere of Skymoot, steering clear of the usual tourist areas and, you realize, of the local law enforcement. As you're putting down in a clearing halfway up a mountain, you hear an otherworldly screech and your ship lurches violently to one side. You would have been thrown to the floor if you hadn't been strapped in.`
```

Reviewer 2: Ignore this as well.

					`	The men load into a shuttle and Karengo directs you down through the atmosphere of <planet>, steering clear of the usual tourist areas and, you realize, of the local law enforcement. As you're putting down in a clearing halfway up a mountain, you hear an otherworldly screech and your ship lurches violently to one side. You would have been thrown to the floor if you hadn't been strapped in.`
					`	"Look lively, lads!" shouts Karengo, the only person still standing. "Your chariot awaits!" He double-checks the jetpacks on the backs of each of the men, or "Dragonriders", as they creatively branded themselves during their first night of the voyage, then opens the side hatch and hurls the Dragonriders out the door before jumping himself.`

> 					`	After the group has left, you land - this time undisturbed - and pull up every external camera you have to watch the action.`

Reviewer 2: ```suggestion
					`	The dragons of Skymoot you learned about on the "Xenofauna and Flora" channel were different from the dragons of mythology, but the beasts you see today aren't quite what you learned about, either. For one, the creatures flying outside your window are largely invisible - a trick of the way the light reflects around the scales on their hides. They are faster, too, or perhaps the ones on the show had been drugged to make them easier to film.`
```

Reviewer 2: And this.

			`	After the group has left, you land - this time undisturbed - and pull up every external camera you have to watch the action.`

> 			`	The dragons of Skymoot you learned about on the "Xenofauna and Flora" channel were different from the dragons of mythology, but the beasts you see today aren't quite what you learned about, either. For one, the creatures flying outside your window are largely invisible - a trick of the way the light reflects around the scales on their hides. They are faster, too, or perhaps the ones on the show had been drugged to make them easier to film.`

Reviewer 3: I don't think it's wise to give Skymoot dragons cloaking bodies.

			`	Suddenly, proximity sensors go off on the starboard side and a camera shows something - or someone - bounce off your shield and tumble down the mountain.`
			choice
				`	(Announce that I'm leaving).`
					goto leave

> 				`	(Stay and watch the Dragonriders).`

Reviewer 2: ```suggestion
```

Author: Excuse my ignorance, but why should this line be deleted?

					goto watch
				`	(Take off and leave everybody else behind).`

> 					goto betrayal

Reviewer 2: ```suggestion
```

Author: Why should this line be deleted?

Reviewer 2: Huh, I put this suggestion on the `goto watch` line, and the next one on the `label watch` line, because the watch choice will go straight into that, but these and at least one other comment have apparently been assigned to different lines.

I wonder if this might be related to changes being made in the time between I started the review and ended it.

Author: Sorry, that might've been it. To be honest I didn't expect to get a review so soon.

			choice
				`	(Take off).`
					goto takeoff
				`	(Swear and take off).`
				`	"Pignuts," you mutter to yourself.`
			label takeoff
			`	The ship takes off quickly enough to make you light-headed. "Get on board!" shouts one of the men over the external speakers as you position the ship just below Karengo and lower the shields. Scales bounce off the roof like diamonds. Karengo releases his grip on the dragon and free falls toward you, slowing at the last possible moment before dropping through the top hatch. You flick on the shields and hit the engines, hard. They respond enthusiastically, rocketing you away from the mountain and into the safety of patrolled airspace. From the cabin comes the sound of back-slapping and, surprisingly, a sob.`

> 	on complete

Reviewer 2: ????

Author: Could you elaborate?

Reviewer 2: This was meant to go on the next line, I think.

> 	on complete

Reviewer 4: ```suggestion
	on complete
		payment 12000 500
```

		conversation

> 			`	As you land and fuel up, Karengo herds the seven surviving men out of the ship. "Meet us at the bar, yeah?" he says to you. "This day deserves a celebration." You nod, but can't help but notice red eyes and wet cheeks among the rest of the Dragonriders as they shuffle off.`

Reviewer 3: ```suggestion
			`As you land and fuel up, Karengo herds the seven surviving men out of the ship. "Meet us at the bar, yeah?" he says to you. "This day deserves a celebration." You nod, but can't help but notice red eyes and wet cheeks among the rest of the Dragonriders as they shuffle off.`
```

			`	Later that day, you spot the Dragonriders at a corner table of the bar. The men, with the exception of Karengo, appear subdued.`

> 			`	"How goes it, <last>?" he asks. "We were just raising a glass to mission success - made possible by you, of course. Great flying, brother." The bartender arrives with a round of drinks and Karengo raises one. "To the Dragonriders!" he says.`

Reviewer 3: The player character is assumed genderless in conversations.
```suggestion
			`	"How goes it, <last>?" he asks. "We were just raising a glass to mission success - made possible by you, of course. Great flying, Captain." The bartender arrives with a round of drinks and Karengo raises one. "To the Dragonriders!" he says.`
```

			choice
				`	"To the Dragonriders!"`
				`	"To our fallen friends!"`
				`	"I'm good, thanks."`
			`	Karengo gives you an unreadable look and lowers his glass.`

> 			`	"Your payment, plus Bolek and Jarek's signup fees. <payment>."`

Reviewer 4: ```suggestion
			`	"Your payment, plus Bolek and Jarek's signup fees: <payment>."`
```

> 			action
> 				payment 12000 500

Reviewer 4: ```suggestion
```

event "fumbling mugger followup timer"

mission "Dangerous Games 1"
	name "Dragonriders"

> 	description "A lively man named Karengo and a few passionate youths are traveling with you to see the dragons on <stopover>. Take the group to <stopover> and return the group back to <destination> once they've finished."

Reviewer 2: No need to say `<stopover>` twice, and repeating `the group` is kind of clunky.
```suggestion
	description "A lively man named Karengo and a few passionate youths are traveling with you to see the dragons on <stopover>. Take the group there, then return them to <destination> once they've finished."
```

			choice
				`	(Head over to "Mr. Karengo").`
				`	(Leave).`
					decline
			`	As you make your way over, you note that nearly everyone in the bar has the same drink you do. In the middle of the crowd, a large, heavily muscled, and tattooed man is gesticulating wildly, apparently telling a story.`
			choice
				`	(Go listen to the man).`
				`	(Leave).`
					decline
			`	"...so then he asks me, have you ever ridden one? And I look back at this crazy joithead and I say, you're a crazy joithead. But he's got me thinking, y'know? Nobody has ever ridden one. Least, not nobody that could talk after." He lets out a sharp laugh, then raises his voice. "So I ask you all; who wants to join Karengo to make history? Who wants to ride a dragon?"`
			`	You've heard the stories of the dragons flying through <planet>'s skies as a child. Great beasts; graceful and dangerous, the apex predators of the air; they are strikingly similar to the dragons described in ancient legends. <planet> is host to many dangerous creatures, and these living myths hold a place at the top of that list.`

> 			`	A cheer goes up from the men, especially the drunk ones. Karengo starts to hand out death and accident waivers and collect money for the expedition. Suddenly, his expression darkens as he cocks his head, listening to an earpiece. "Listen up!" he yells, "Seems my regular pilot got in the way of some less reputable folk, and my replacement pilot misplaced his balls this evening and can't find 'em! We need ourselves a ship and a captain to bring this lot to <planet>! Who's brave enough? Who else wants to make history?"`

Reviewer 2: ```suggestion
			`	A cheer goes up from the men, especially the drunk ones. Karengo starts to hand out death and accident waivers and collect money for the expedition. Suddenly, his expression darkens as he cocks his head, listening to an earpiece. "Listen up!" he yells, "Seems my regular pilot got in the way of some less reputable folk, and my replacement pilot misplaced his balls this evening and can't find 'em! We need ourselves a ship and a captain to bring this lot to <stopovers>! Who's brave enough? Who else wants to make history?"`
```

			choice
				`	"Captain <last>, at your service. We leave at dawn!"`
				`	(Stay silent)."`
					decline
			`	You wake up significantly after dawn with empty bottles and unconscious men strewn all over the ship's mess hall. Karengo peels himself off the floor and gives you a crooked smile, shielding his eyes from the light. "Alright, <last>" he says, "let's do this."`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying the passengers hasn't entered this system yet. Better depart and wait for it.`
	on stopover
		conversation

> 			`	Karengo directs you down through the atmosphere of <planet>, steering clear of the usual tourist areas and, you realize, of the local law enforcement. As you're putting down in a clearing halfway up a mountain, you hear an otherworldly screech and your ship lurches violently to one side. You would have been thrown to the floor if you hadn't been strapped in.`

Reviewer 2: First line in the conversation shouldn't have an indent, I think.
```suggestion
			`Karengo directs you down through the atmosphere of Skymoot, steering clear of the usual tourist areas and, you realize, of the local law enforcement. As you're putting down in a clearing halfway up a mountain, you hear an otherworldly screech and your ship lurches violently to one side. You would have been thrown to the floor if you hadn't been strapped in.`
```

			`	"Look lively, lads!" shouts Karengo, the only person still standing. "Your chariot awaits!" He double-checks the jetpacks on the backs of each of the men, or "Dragonriders", as they creatively branded themselves during their first night of the voyage, then opens the side hatch and hurls the Dragonriders out the door before jumping himself.`
			`	After the group has left, you land - this time undisturbed - and pull up every external camera you have to watch the action.`

> 			`	The dragons of <planet> you learned about on the "Xenofauna and Flora" channel were different from the dragons of mythology, but the beasts you see today aren't quite what you learned about, either. For one, the creatures flying outside your window are largely invisible - a trick of the way the light reflects around the scales on their hides. They are faster, too, or perhaps the ones on the show had been drugged to make them easier to film.`

Reviewer 2: ```suggestion
			`	The dragons of Skymoot you learned about on the "Xenofauna and Flora" channel were different from the dragons of mythology, but the beasts you see today aren't quite what you learned about, either. For one, the creatures flying outside your window are largely invisible - a trick of the way the light reflects around the scales on their hides. They are faster, too, or perhaps the ones on the show had been drugged to make them easier to film.`
```

			`	A quick survey of the scene shows that Karengo has tagged three dragons with some sort of harmless marker gun to make them easier to track. Eight of the Dragonriders are chasing the largest one, which is missing part of a tooth and has blackened scales from where it hit your shuttle's shield. The last Dragonrider is nowhere in sight.`
			`	Suddenly, proximity sensors go off on the starboard side and a camera shows something - or someone - bounce off your shield and tumble down the mountain.`
			choice
				`	(Announce that I'm leaving).`
					goto leave
				`	(Stay and watch the Dragonriders).`

> 					goto watch

Reviewer 2: ```suggestion
```

				`	(Take off and leave everybody else behind).`
					goto betrayal

> 			label watch

Reviewer 2: ```suggestion
```

			`	A cheer goes up from the men, especially the drunk ones. Karengo starts to hand out death and accident waivers and collect money for the expedition. Suddenly, his expression darkens as he cocks his head, listening to an earpiece. "Listen up!" he yells, "Seems my regular pilot got in the way of some less reputable folk, and my replacement pilot lost his balls this evening and can't find 'em! We need ourselves a ship and a captain to bring this lot to <stopovers>! Who's brave enough? Who else wants to make history?"`
			choice
				`	"Captain <last>, at your service. We leave at dawn!"`
					goto accept
				`	(Stay silent.)`
			`	You remain silent as Karengo surveys the crowd. "What, nobody with a ship has the cojones to make history? I didn't know this bar was full of a bunch of sissies."`
			`	One captain in the back of the bar jumps up. "Mama didn't raise no sissy! I'll bring you all to Skymoot." A cheer rises up from the crowd and the captain is treated to several rounds of drinks. After the captain and all the passengers are sufficiently drunk, they saunter their way out of the bar, drunkenly singing about adventure as they go.`
				decline
			label accept

> 			`	After a cheer rises up from the crowd, you're treated to several rounds of drinks by your passengers; you sense any attempt to refuse would be ignored. You're unsure of how much you drank, but it was enough to make sure that you didn't remember. You wake up significantly after dawn with empty bottles and unconscious men strewn all over the ship's mess hall. Karengo peels himself off the floor and gives you a crooked smile, shielding his eyes from the light. "Alright, <last>," he says, "let's do this."`

Reviewer 5: ```suggestion
			`	After a cheer rises up from the crowd, and you're treated to several rounds of drinks by your passengers; you sense any attempt to refuse would be ignored. You're unsure of how much you drank, but it was enough to make sure that you didn't remember. You wake up significantly after dawn with empty bottles and unconscious men strewn all over the ship's mess hall. Karengo peels himself off the floor and gives you a crooked smile, shielding his eyes from the light. "Alright, <last>," he says, "let's do this."`
```

Author: I'm not really sure what the `and` is for in this case.

Reviewer 5: Two independent clauses. Need either a period between or a comma with a coordinating conjunction (`and`).

> 			label accept
> 			`	After a cheer rises up from the crowd, you're treated to several rounds of drinks by your passengers; you sense any attempt to refuse would be ignored. You're unsure of how much you drank, but it was enough to make sure that you didn't remember. You wake up significantly after dawn with empty bottles and unconscious men strewn all over the ship's mess hall. Karengo peels himself off the floor and gives you a crooked smile, shielding his eyes from the light. "Alright, <last>," he says, "let's do this."`
> 				accept

Reviewer 5: ```suggestion
			label accept
			`	After a cheer rises up from the crowd, you're treated to several rounds of drinks by your passengers; you sense any attempt to refuse would be ignored. You're unsure of how much you drank, but it was enough to make sure that you didn't remember. You wake up significantly after dawn with empty bottles and unconscious men strewn all over the ship's mess hall. Karengo peels himself off the floor and gives you a crooked smile, shielding his eyes from the light. "Alright, <last>," he says, "let's do this."`
				accept
			label money
			`	The crowd's eyes turn to you. Karengo squints, then laughs. "Ha! What kind of question is that? Of course you're getting paid. <payment> to bring myself and these other fine folks to <planet> and back - <bunks> of us total. What say you?" The crowd watches you with eager anticipation."`
			choice
				`	"Works for me. We leave tomorrow."`
					goto accept
				`	"<payment> won't be enough. I'm out."`
			`	You're treated to some jeering and rude looks as you leave the bar.`
				decline
```

	on visit
		dialog `You land on <planet>, but realize that your escort carrying the passengers hasn't entered this system yet. Better depart and wait for it.`
	on stopover
		conversation

> 			`Karengo directs you down through the atmosphere of Skymoot, steering clear of the usual tourist areas and, you realize, of the local law enforcement. As you're putting down in a clearing halfway up a mountain, you hear an otherworldly screech and your ship lurches violently to one side. You would have been thrown to the floor if you hadn't been strapped in.`

Reviewer 5: ```suggestion
			`Karengo directs you down through the atmosphere of Skymoot, steering clear of the usual tourist areas and, you realize, of the local law enforcement. As you're putting down in a clearing halfway up a mountain, you hear an otherworldly screech as your ship lurches violently to one side. You would have been thrown to the floor if you hadn't been strapped in.`
```

			choice
				`	(Announce that I'm leaving.)`
					goto leave
				`	(Stay and watch the Dragonriders.)`
				`	(Take off and leave everybody else behind.)`
					goto betrayal
			`	Checking your camera feeds again, you see that object rolling down the hill is indeed a body. On another monitor, one of the Dragonriders is keeping just above a dragon, attempting to lower himself down. The dragon rolls in midair, its wing catching the man and sending him flying out of control. Fortunately, a leafy tree branch prevents any further physical harm. Meanwhile, a second dragon is pursuing another one of the Dragonriders.`
			choice
				`	(Get everyone out of there.)`
					goto leave
				`	(Take off and leave everybody else behind.)`
			label betrayal
			action
				fail
			`	The ship takes off with a roar at your touch. Between the cries of the dragons, the sound of the engines, and the air rushing past, you can barely hear the fear and betrayal in the voices of the men you've left behind.`
				decline
			label leave
			`	You turn on the external speakers. "We are leaving," you blast to the survivors. "Get on board."`

> 			`	The defeated Dragonriders need no further encouragement. They beeline for the hatch and you flicker the shield to let them on deck, where they collapse, wide-eyed and panting. Karengo isn't with them. You turn to ask about him, when a flicker of motion on a monitor catches your eye.`

Reviewer 5: ```suggestion
			`	The defeated Dragonriders need no further encouragement. They beeline for the hatch and you flicker the shield to let them on deck, where they collapse, wide-eyed and panting. Karengo isn't with them. You turn to ask about him when a flicker of motion on a monitor catches your eye.`
```

	on complete
		payment 12000 500
		conversation
			`As you land and fuel up, Karengo herds the seven surviving men out of the ship. "Meet us at the bar, yeah?" he says to you. "This day deserves a celebration." You nod, but can't help but notice red eyes and wet cheeks among the rest of the Dragonriders as they shuffle off.`
			`	Later that day, you spot the Dragonriders at a corner table of the bar. The men, with the exception of Karengo, appear subdued.`
			`	"How goes it, <last>?" he asks. "We were just raising a glass to mission success - made possible by you, of course. Great flying, Captain." The bartender arrives with a round of drinks and Karengo raises one. "To the Dragonriders!" he says.`
			choice
				`	"To the Dragonriders!"`
				`	"To our fallen friends!"`
				`	"I'm good, thanks."`

> 			`	Karengo gives you an unreadable look, lowers his glass, and says, "Your payment, plus Bolek and Jarek's signup fees: <payment>."`

Reviewer 5: Only one guy died, right?

Author: Fixed, thanks for catching that.
